### PR TITLE
replaced html comments with blade comments

### PR DESCRIPTION
**Pull request by alashow**

### CODE REVIEW

File: src/CrudPanel.php [hunk 1]
```diff
@@ -61,6 +61,9 @@ class CrudPanel
     public $db_column_types = [];
     public $default_page_length = false;
 
+    // extra columns to query when using ajax tables
```
Comment (lloy0076) [inline src/CrudPanel.php]: Is this relevant to the PULL description?

File: src/PanelTraits/Read.php [hunk 2]
```diff
@@ -87,8 +87,10 @@ public function getFields($form, $id = false)
     /**
      * Check if the create/update form has upload fields.
      * Upload fields are the ones that have "upload" => true defined on them.
+     *
```
Comment (lloy0076) [inline src/PanelTraits/Read.php]: Is this relevant to the PULL description?

File: src/PanelTraits/Read.php [hunk 3]
```diff
@@ -87,8 +87,10 @@ public function getFields($form, $id = false)
...
      * @param  [form] create / update / both - defaults to 'both'
      * @param  [id] id of the entity - defaults to false
+     *
```
Comment (lloy0076) [inline src/PanelTraits/Read.php]: Is this relevant to the PULL description?

File: src/PanelTraits/Read.php [hunk 4]
```diff
@@ -162,6 +163,7 @@ public function enableAjaxTable()
 
     /**
      * Check if ajax is enabled for the table view.
+     *
```
Comment (lloy0076) [inline src/PanelTraits/Read.php]: Is this relevant to the PULL description?

File: src/PanelTraits/Read.php [hunk 5]
```diff
@@ -171,7 +173,9 @@ public function ajaxTable()
 
     /**
      * Get the HTML of the cells in a table row, for a certain DB entry.
+     *
```
Comment (lloy0076) [inline src/PanelTraits/Read.php]: Is this relevant to the PULL description?

File: src/PanelTraits/Read.php [hunk 6]
```diff
@@ -171,7 +173,9 @@ public function ajaxTable()
...
      * @param  Entity $entry A db entry of the current entity;
+     *
```
Comment (lloy0076) [inline src/PanelTraits/Read.php]: Is this relevant to the PULL description?

File: src/PanelTraits/Read.php [hunk 7]
```diff
@@ -186,8 +190,10 @@ public function getRowViews($entry)
 
     /**
      * Get the HTML of a cell, using the column types.
-     * @param  array $column
-     * @param  Entity $entry   A db entry of the current entity;
+     *
```
Comment (lloy0076) [inline src/PanelTraits/Read.php]: Is this relevant to the PULL description?

File: src/PanelTraits/Read.php [hunk 8]
```diff
@@ -207,6 +213,26 @@ public function getCellView($column, $entry)
...
+    {
+        array_push($this->ajax_extra_fields, $field);
+    }
+
+    /**
+     * Extra fields/columns name to include in select query when using ajax tables
+     *
+     * @return array string array of field/column names
+     */
+    public function getAjaxExtraFields()
```
Comment (lloy0076) [inline src/PanelTraits/Read.php]: Isn't this another **function**?

File: src/PanelTraits/Read.php [hunk 9]
```diff
@@ -207,6 +213,26 @@ public function getCellView($column, $entry)
         }
     }
 
+    /**
+     * Add field/column name to include in select query when using ajax tables
+     *
+     * @param string $field field/column name
+     */
+    public function addAjaxExtraField($field)
```
Comment (lloy0076) [inline src/PanelTraits/Read.php]: Is this adding to some other non-merged pull - the pull description says something about Blade comments and this is another function...

File: src/PanelTraits/Read.php [hunk 10]
```diff
@@ -207,6 +213,26 @@ public function getCellView($column, $entry)
         }
     }
 
+    /**
+     * Add field/column name to include in select query when using ajax tables
```
Comment (lloy0076) [inline src/PanelTraits/Read.php]: Full stop at end?

File: src/PanelTraits/Read.php [hunk 11]
```diff
@@ -207,6 +213,26 @@ public function getCellView($column, $entry)
...
+     *
+     * @param string $field field/column name
+     */
+    public function addAjaxExtraField($field)
+    {
+        array_push($this->ajax_extra_fields, $field);
+    }
+
+    /**
+     * Extra fields/columns name to include in select query when using ajax tables
```
Comment (lloy0076) [inline src/PanelTraits/Read.php]: Full stop at end?

File: src/resources/views-elfinder/elfinder.blade.php [hunk 12]
```diff
@@ -1,25 +1,25 @@
 @extends('backpack::layout')
 
 @section('after_scripts')
-    <!-- jQuery and jQuery UI (REQUIRED) -->
+    {{-- jQuery and jQuery UI (REQUIRED) --}}
```
Comment (lloy0076) [inline src/resources/views-elfinder/elfinder.blade.php]: Actually, why? This is a comment about the HTML and not about the PHP!

File: src/resources/views-elfinder/elfinder.blade.php [hunk 13]
```diff
@@ -1,25 +1,25 @@
...
     <link rel="stylesheet" href="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/themes/smoothness/jquery-ui.css" />
-    <!-- <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script> -->
+    {{-- <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script> --}}
```
Comment (lloy0076) [inline src/resources/views-elfinder/elfinder.blade.php]: It's either one or the other - personally I'd just **delete** the relevant one here.

File: src/resources/views-elfinder/elfinder.blade.php [hunk 14]
```diff
@@ -1,25 +1,25 @@
...
     <script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js"></script>
 
-    <!-- elFinder CSS (REQUIRED) -->
+    {{-- elFinder CSS (REQUIRED) --}}
```
Comment (lloy0076) [inline src/resources/views-elfinder/elfinder.blade.php]: That's either a redundant comment OR it should remain (it's a comment about the HTML page, not about the logic on the page).

**Comment by alashow** [comment]
@lloy0076 Hey. I'm sorry but I thought this PR was closed and I'm just using the blade-comments branch as my own version. That's why I merged last 3 commits into blade-comments. 
Replacing HTML comments to blade comments was discussed in [here](https://github.com/Laravel-Backpack/Base/pull/129). 

Sorry for the confusion.
